verification/src/compiler/version.rs

```diff
@@ -27,15 +23,14 @@ impl FromStr for ReleaseVersion {
     type Err = ParseError;
 
     /// Parses release version from string formated as
-    /// `solc-v*VERSION*+commit.*COMMITHASH*`, example
-    /// `solc-v0.8.9+commit.e5eed63a`
+    /// `*PREFIX**VERSION*+commit.*COMMITHASH*`, examples:
+    /// `v0.8.9+commit.e5eed63a`
+    /// `0.8.4+commit.dea1b9ec`
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        let (version_str, commit_hash) = sscanf::scanf!(s, "solc-v{}+commit.{}", String, String)
-            .map_err(|e| ParseError::Parse(format!("{:?}", e)))?;
-        let version = Version::from_str(&version_str).map_err(ParseError::SemVer)?;
-        if !version.pre.is_empty() || !version.build.is_empty() {
-            return Err(ParseError::VersionFormat(version_str));
-        }
+        let (_prefix, major, minor, patch, commit_hash) =
+            sscanf::scanf!(s, "{String:/v?/}{}.{}.{}+commit.{}", u64, u64, u64, String)
```

Comment:
Nit: why not to move String part out of the brackets?

Reply:
And it would be nice to add a comment which specifies that for now we support only `v` or empty string as a prefix